src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -577,6 +577,48 @@ private void testEditInstructorAction() throws Exception {
                 sectionToCheck, sessionToCheck, InstructorCourseEditPage.SESSION_MODIFY_RESPONSES));
 
         courseEditPage.selectRoleForInstructor(editInstructorIndex, "Co-owner");
+
+        ______TS("Verify that one instructor must be visible to students");
+        // Need to prepare for test by setting all instructors as not displayed to students
+        courseEditPage.editInstructor(1, "New name", "new_email@email.tmt",
+                false, "Instructor", "Custom");
+        courseEditPage.editInstructor(2, "New name", "InsCrsEdit.instructor@gmail.tmt",
+                false, "New display name", "Co-owner");
+        courseEditPage.editInstructor(3, "New name", "InsCrsEdit.instructor@gmail.tmt",
+                false, "Instructor", "Co-owner");
+        courseEditPage.editInstructor(4, "Teammates New Instructor", "InsCrsEdit.Helper@gmail.tmt",
+                false, "Instructor", "Custom");
+        courseEditPage.editInstructor(5, "Teammates New Instructor", "InsCrsEdit.newInstr@gmail.tmt",
+                false, "Instructor", "Custom");
+        courseEditPage.editInstructor(6, "Teammates Observer", "InsCrsEdit.observer@gmail.tmt",
+                false, "Instructor", "Observer");
+        courseEditPage.editInstructor(7, "Teammates Test", "InsCrsEdit.test@gmail.tmt",
+                false, "Instructor", "Co-owner");
+        courseEditPage.editInstructor(8, "Teammates Do Not Save", "InsCrsEdit.tutor@gmail.tmt",
+                false, "Instructor", "Tutor");
```

Comment:
This code block can be made less fragile. Extract into a `makeOtherInstructorsInvisible(courseId, instructorId)`. The method should iterate though all instructors and make them invisible unless it is the specified instructor.